package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "dummy_package",
```

Comment:
> Is this necessary? Seeing [tier4/pre-commit-hooks-ros#20 (comment)](https://github.com/tier4/pre-commit-hooks-ros/pull/20#issuecomment-1049830255), it's for Prettier and I think we don't use Prettier.

I see. Then, I will remove this file.

Reply:
fbcd66f

Reply:
Did it work correctly?

Reply:
Yes.

I modified `pre-commit-config.yaml`  for testing:
```
  - repo: https://github.com/KeisukeShima/autoware-spell-check-dict
    rev: fbcd66fa6b3cf564062c4d2db01e13bf5bfbfb2a
    hooks:
      - id: sort-cspell-json
```

And the result is:
```
❯ pre-commit try-repo . -a -v
[WARNING] Creating temporary repo with uncommitted changes...
[INFO] Initializing environment for /tmp/tmpdcczijjr/shadow-repo.
===============================================================================
Using config:
===============================================================================
repos:
-   repo: /tmp/tmpdcczijjr/shadow-repo
    rev: 3adf84249ed830bb6ea3541aeda57c24955ef327
    hooks:
    -   id: sort-cspell-json
===============================================================================
[INFO] Installing environment for /tmp/tmpdcczijjr/shadow-repo.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
sort cspell.json.........................................................Passed
- hook id: sort-cspell-json
- duration: 0.14s
- ```